{
  "name": "fuse-error-example",
  "version": "1.0.0",
  "description": "This repo illustrates an error with JS calling TS",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/wasd171/fuse-error-example.git"
  },
  "author": "wasd171",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/wasd171/fuse-error-example/issues"
  },
  "homepage": "https://github.com/wasd171/fuse-error-example#readme",
  "dependencies": {
    "babel-core": "^6.25.0",
    "babel-preset-env": "^1.5.2",
    "fuse-box": "^2.1.0",
    "typescript": "^2.4.1"
  }
}
